pom.xml

```diff
@@ -86,5 +86,35 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
-
-</project>
\ No newline at end of file
+    <build>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-surefire-plugin</artifactId>
+                    <version>3.0.0-M6</version>
+                </plugin>
+            </plugins>
+        </pluginManagement>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <version>3.3.0</version>
+                <executions>
+                    <execution>
+                        <id>unpack-schemas</id>
+                        <goals>
+                            <goal>unpack</goal>
+                        </goals>
+                        <configuration>
+                            <artifact>org.apache.kafka:kafka-clients:3.1.0</artifact>
+                            <includes>common/message/*.json</includes>
+                            <outputDirectory>${project.build.directory}/message-schemas</outputDirectory>
```

Comment:
So the idea would be to pick up the json from a released version of Kafka. Which is neat, but would make it more difficult to work on filters for new RPCs (or new versions of existing RPCs) that aren't in a release. That could actually be useful at some point in the future. An alternative might be to access the json from github via HTTP, but that would prevent disconnected builds, which isn't great either.

Reply:
Yeah, so my thinking is to leave that aspect of how schemas are obtained to the user of the plug-in. In that light, I could have left this as a sub-module, but changing it to the dependency plug-in seemed easy enough. I.e. also in the kproxy build, we'd configure the dependency plug-in for obtaining a released version, or any other means we consider fit for the job.